distributions/osgi/replication.policy

```diff
@@ -1,5 +1,17 @@
 priority "grant";
 
+grant codeBase "file:/ddf-adapter" {
+    permission java.lang.RuntimePermission "createClassLoader";
+}
+
+grant codeBase "file:/ddf-adapter/replication-api-impl" {
+    permission java.lang.RuntimePermission "createClassLoader";
+}
+
+grant codeBase "file:/org.apache.servicemix.bundles.xstream" {
+    permission java.lang.RuntimePermission "createClassLoader";
+}
+
```

Comment:
I needed to add the following grant to get this working on DDF
```
grant codeBase "file:/replication-api-impl" {
    permission java.io.FilePermission "${ddf.home.perm}etc${/}keystores${/}serverKeystore.jks", "read";
}
```
Would you mind adding it here?

Reply:
Roger, on it